x-pack/legacy/plugins/uptime/server/rest_api/snapshot/get_snapshot_count.ts

```diff
@@ -22,15 +22,25 @@ export const createGetSnapshotCount: UMRestApiRouteCreator = (libs: UMServerLibs
   options: {
     tags: ['access:uptime'],
   },
-  handler: async (_context, request, response): Promise<any> => {
+  handler: async (
+    {
+      core: {
+        elasticsearch: {
+          dataClient: { callAsCurrentUser },
```

Comment:
Some thought here with the boilerplate

Reply:
Answered above

Reply:
https://github.com/elastic/kibana/pull/51403#discussion_r355709876